gtsfm/loader/astronet_loader.py

```diff
@@ -0,0 +1,231 @@
+"""Simple loader class that reads a dataset with metadata formatted in the COLMAP style.
+
+Authors: Travis Driver
+"""
+
+import os
+from pathlib import Path
+from typing import Optional, Dict, Tuple, List
+
+from gtsam import Cal3Bundler, Pose3, Rot3, Point3, SfmTrack
+
+import gtsfm.utils.io as io_utils
+import gtsfm.utils.logger as logger_utils
+from gtsfm.common.image import Image
+from gtsfm.loader.loader_base import LoaderBase
+
+from thirdparty.colmap.scripts.python.read_write_model import read_model
+from thirdparty.colmap.scripts.python.read_write_model import Camera as ColmapCamera
+from thirdparty.colmap.scripts.python.read_write_model import Image as ColmapImage
+from thirdparty.colmap.scripts.python.read_write_model import Point3D as ColmapPoint3D
+
+
+logger = logger_utils.get_logger()
+
+
+class AstroNetLoader(LoaderBase):
```

Comment:
Should the class be called `AstronetLoader`, considering the context here? The file is also called astronet_loader.

Reply:
My dataset is called `AstroNet`,  so yeah I think it should be `AstronetLoader` instead.

Reply:
Akshay did you mean disambiguating from a CNN architecture/network name here?

Reply:
Yeah I just meant using a small 'n' instead of 'N' because Astronet is one thing as far as the loader is concerned. Its resolved now.